pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -28,3 +28,13 @@ func (d *DualWriterMode1) Create(ctx context.Context, obj runtime.Object, create
 
 	return legacy.Create(ctx, obj, createValidation, options)
 }
+
+// Get overrides the default behavior of the DualWriter and reads only to LegacyStorage.
+func (d *DualWriterMode1) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	legacy, ok := d.legacy.(rest.Getter)
+	if !ok {
+		return nil, fmt.Errorf("legacy storage rest.Get is missing")
+	}
+
```

Comment:
I don't think we actually need this, since the local Storage and LegacyStorage interfaces both embed rest.Getter. We do need it for the other methods that aren't part of the baseline interface though.